.github/workflows/examples.yml

```diff
@@ -0,0 +1,106 @@
+name: Examples
+
```

Comment:
Unfortunately as this appears in the status SVG badge we should rename this for now.
I suggest renaming it to `Build`